lib/irc-view.coffee

```diff
@@ -51,12 +52,14 @@ class IrcView extends ScrollView
 
     msg = $('<span>')
     un.addClass 'msg'
+    autolinker = new Autolinker(newWindow: false)
```

Comment:
@aaronholla This instantiation of the autolinker object should go in the `initialize` method, otherwise we'll be instantiating a new object for each message.